src/quickPick.ts

```diff
@@ -8,13 +8,15 @@ const getBaseCopyOptions = (editor: TextEditor): QuickPickOption[] => {
 		relativeWithLine,
 		absolutePath,
 		fileName,
-		workspacePath
+		absolutePathWithLine,
+		workspacePath,
 	} = getDescriptions(editor)
 
 	return [
 		{ label: '$(code) Path with line number', description: relativeWithLine },
 		{ label: '$(code) Path from content root', description: relativePath },
 		{ label: '$(code) Absolute path', description: absolutePath },
+		{ label: '$(code) Absolute path with line number', description: absolutePathWithLine },
```

Comment:
A tricky thing with the code system is.
People will start memorising the code they use often.

Adding one in the middle will cause problems for the users.
Since there are not too many users of the extension yet, this should be okay. But perhaps is an indication of a missing framework to make it extensible.